Lombiq.Hosting.Tenants.IdleTenantManagement.Tests.UI/Extensions/IdleTenantManagementExtensions.cs

```diff
@@ -26,5 +26,5 @@ public static class IdleTenantManagementExtensions
     public static readonly Func<IWebApplicationInstance, Task> AssertAppLogsWithIdleCheckAsync =
         async webApplicationInstance =>
             (await webApplicationInstance.GetLogOutputAsync())
-            .ShouldContain("Shutting down tenant \"Default\" because of idle timeout.");
+            .ShouldContain("Shutting down tenant \"IdleTenantShutdown\" because of idle timeout.");
```

Comment:
Shouldn't the tenant name be an input parameter also like it is in the TestIdleTenantManagerBehaviorAsync? Otherwise, this might fail.

Reply:
Imo not really because in the tenant setup of the UITestExtension method this exact name is give for the tenant to be created. So in theory it can't fail. This is just a static way of testing a tenant is indeed shutdown or not. I don't think we need flexible testing.

Reply:
Yeah, but then why do you have the option for the test itself to be anything?